src/lib.rs

```diff
@@ -2,7 +2,7 @@ extern crate alloc;
 
 pub mod api;
 #[allow(dead_code)]
-pub mod css3_parser;
+pub mod css;
```

Comment:
I like this change.  Hopefully we can make similar changes elsewhere in the project.

Reply:
Is your plan to pick up other versions of CSS?  Would `pub mod css3;` be better?

Reply:
Is css3 compatible with css2  (ie: can a css2 stylesheet be loaded by a css3 parser)?

Reply:
I suspect we would want separate `css3::` and `css2::` Rust modules at some point, possibly with a shared trait to serve as the interface.